Keyboards/cmake.bash

```diff
@@ -147,11 +93,72 @@ esac
 
 # Append generator name (to support building both types on the same system)
 BuildPath="${BuildPath}.${MAKE}"
-echo "Selected Generator: ${CMAKE_GENERATOR}"
-
-
 # Prepend OSType (so not to clobber builds if using the same storage medium, i.e. dropbox)
 BuildPath="${OSTYPE}.${BuildPath}"
+
```

Comment:
Instead of having to iterate through all the arguments, which while useful, tends to get unwieldly.

```bash
DefaultMap=${DefaultMap:-${DefaultMapOverride}}
PartialMapsExpanded=${PartialMapsExpanded:-${PartialMapsExpandedOverride}}
``` 

Then just set those variables when you call the build script.

Reply:
Bah, that's backwards, one sec.

Reply:
```bash
DefaultMap=${DefaultMapOverride:-${DefaultMap}}
PartialMapsExpanded=${PartialMapsExpandedOverride:-${PartialMapsExpanded}}
``` 

That's better...